### PR TITLE
DF-80: Implement search term highlighting for results 

### PR DESCRIPTION
**Pull request by AshTNA**
Hi @ababic 

Would you be able to merge this PR if all is OK?

Thanks :+1:

### CODE REVIEW

**Review by ababic — CHANGES_REQUESTED** [review]
Thanks for the quick turnaround @AshTNA. I think we could simplify things somewhat by creating a 'filter' instead, though

**Comment by AshTNA** [comment]
@ababic 

Will do :+1:

File: etna/search/templatetags/search_tags.py [hunk 1]
```diff
@@ -84,6 +109,15 @@ def record_score(record) -> str:
     return record.get("_score")
 
 
+@register.filter
+def search_result_striptags(raw_html) -> str:
+    """
+    Takes a string of HTML and strips out all tags apart from <mark> for search results highlighting.
+    """
+    print(raw_html)
```
Comment (ababic) [inline etna/search/templatetags/search_tags.py:117]: @AshTNA Remember to tidy up :)

Comment (AshTNA) [reply etna/search/templatetags/search_tags.py]: @ababic Whoops!

**Review by ababic — APPROVED** [review]
Nice work @AshTNA! LGTM

**Pull request merged**